_includes/footer.html

```diff
@@ -30,8 +24,14 @@
           </li>
           {% endif %}
           {% if site.stream_playlist %}
+        </ul>
+      </div>
+
+      <div class="footer-col  footer-col-2">
+        <ul class="social-media-list">
+          <li style="white-space: pre-wrap">{{ site.address }}</li>
           <li>
-            Player not working? <br>
+            <br/>
```

Comment:
Nit: you shouldn't use line breaks to set list item spacing. This should be done using CSS.